com.unity.render-pipelines.universal/Runtime/2D/LightUtility.cs

```diff
@@ -170,6 +170,7 @@ public static Bounds GenerateSpriteMesh(Mesh mesh, Sprite sprite)
             mesh.SetVertexBufferParams(vertices.Length, SpriteLightMeshVertex.VertexLayout);
             mesh.SetVertexBufferData(vertices, 0, 0, vertices.Length);
             mesh.SetIndices(srcIndices, MeshTopology.Triangles, 0, true);
+            mesh.RecalculateBounds();
```

Comment:
why is this needed? the `true` in the SetIndices should recalculate the bounds... or does it not?

Reply:
Removed.